### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by IamManchanda**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - name: Uses Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: npm ci
+      - run: npm run format
+      - run: npm run format:check
+      - run: npm test -- --coverage
+        env:
+          CI: true
```
Comment (IamManchanda) [inline .github/workflows/ci.yml:19]: Looks good to me

**Review by HarryJamstack — DISMISSED** [review]
Looks Good!

**Review by HarryJamstack — APPROVED** [review]
Looks good, thanks for adding me!

**Review by HarryJamstack — APPROVED** [review]
Looks Good!

**Pull request merged**